water_mark/water_mark.py

```diff
@@ -0,0 +1,303 @@
+            gdal.Warp(
+                output_file,
+                input_file,
+                options=['-cutline', cut_size_shp, '-crop_to_cutline']
+            )
+            delete_junk('inputs')
+
+
+def move_sar_mask_to_out_dir(out_dir):
+    for sar in os.listdir(out_dir):
+        image = os.path.join('inputs', f"{sar}.tif")
+        mask = os.path.join('inputs', f"{sar}Mask.tif")
+        copy_location = os.path.join(out_dir, sar)
+        shutil.copy(image, copy_location)
+        os.rename(
+            os.path.join(out_dir, sar, f"{sar}.tif"),
+            os.path.join(out_dir, sar, f"{sar}_Image.tif")
+        )
+        shutil.copy(mask, copy_location)
+        os.remove(mask)
+
+
+def cut_sar_to_tiles(out_dir, mxm_tile_size):
+    for sar in os.listdir(out_dir):
+        image = os.path.join(out_dir, sar,  f"{sar}_Image.tif")
+        imageData = gdal.Open(image)
+        xStep, yStep = mxm_tile_size, mxm_tile_size
+        xSize, ySize = imageData.RasterXSize, imageData.RasterYSize
+        count = 0
+        for x in range(0, xSize, xStep):
+            for y in range(0, ySize, yStep):
+                # fileName = 'Image_<sar>_<0-9+>.tif'
+                fileName = f"Image_{sar}_{count}.tif"
+                output_file = os.path.join(out_dir, sar, fileName)
+                input_file = image
+                gdal.Translate(
+                    output_file,
+                    input_file,
+                    srcWin=[x, y, xStep, yStep],
+                    format="GTiff"
+                )
+                count += 1
+
+
+def clip_mask_to_sar(out_dir):
+    for sar in os.listdir(out_dir):
+        for tile in os.listdir(os.path.join(out_dir, sar)):
+            cut_size = tile
+            cut_size_shp = f"{cut_size[:-4]}.shp"  # change ext
+            input_file = sar
+            output_file = f"Mask_{tile[5:]}"
+            subprocess.call(f"gdaltindex {cut_size_shp} {cut_size}", shell=True)
+            gdal.Warp(
+                output_file,
+                input_file
+                options=['-cutline', cut_size_shp, '-crop_to_cutline']
+            )
+            delete_junk('inputs')
+
+
+def trim_masks(out_dir, mxm_tile_size):
+    for sar in os.listdir(out_dir):
+        for f in os.listdir(os.path.join(out_dir, sar)):
+            if f == f"{sar}Image.tif" or f == f"{sar}Mask.tif" or f.endswith('.py'):
+                pass
+            else:
+                resizedImage = f"RS{f}"
+                output_file = resizedImage
+                input_file = f
+                gdal.Translate(
+                    output_file,
+                    input_file,
+                    options=["-outsize", mxm_tile_size, mxm_tile_size]
+                )
+                os.remove(os.path.join(out_dir, f))
+                os.rename(resizedImage, resizedImage[2:])  # remove 'RS'
+
+
+def reclassify_mask(out_dir):
+    for sar in os.listdir(out_dir):
+        for f in os.listdir(os.path.join(out_dir, sar)):
```

Comment:
Typically `f` is used for the name of file objects like so:
```python
with open('something') as f:
    pass
```

So it is a little confusing to use it as a file name here

Reply:
changed to f_name